portal-kernel/src/com/liferay/portal/kernel/workflow/DefaultWorkflowTransition.java

```diff
@@ -34,7 +34,11 @@ public String getLabel(Locale locale) {
 			return HtmlUtil.escape(name);
 		}
 
-		String label = _labelMap.get(LocaleUtil.getSiteDefault());
+		String label = null;
+
+		if (MapUtil.isNotEmpty(_labelMap)) {
```

Comment:
@p-albuquerque could you please check only if _labelMap is different of null?

Thanks!